### PR TITLE
Add Hungarian translation

### PR DESCRIPTION
**Pull request by meskobalazs**
I have translated all the strings.

### CODE REVIEW

File: src/_locales/hu/messages.json [hunk 1]
```diff
@@ -55,72 +55,72 @@
         "message": "Firefox for iOS"
```
Comment (jhonatasrm) [inline src/_locales/hu/messages.json]: In Hungarian is used "Firefox for iOS"?

File: src/_locales/hu/messages.json [hunk 2]
```diff
@@ -55,72 +55,72 @@
...
     "clear_notifications": {
-        "message": "Clear notifications"
+        "message": "Értesítések törlése"
     },
     "show_notifications": {
-        "message": "Show browser notifications"
+        "message": "Böngészőértesítések megjelenítése"
     },
     "description_notifications": {
         "message": "Determines whether you will receive notification popups when new questions are found"
```
Comment (jhonatasrm) [inline src/_locales/hu/messages.json]: Can you take a look at that? I think you forgot that one. 

Thank you very much!  🙂

Comment (meskobalazs) [reply src/_locales/hu/messages.json]: Sure

File: src/_locales/hu/messages.json (hunk 1 above)
Comment (meskobalazs) [reply src/_locales/hu/messages.json]: Yes, we treat it as a brand (proper noun).

**Comment by jhonatasrm** [comment]
Thank you very much for your help @meskobalazs ! 

🙂

**Pull request merged**